Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN \
         libjpeg-turbo \
         libpng \
         libstdc++ \
+        yaml-dev \

```

Comment:
-dev are only need for compilation. There should be a - libs equivalent

Reply:
If you need this only for wheels, you have to add this to the wheels CI on core and not base image

Reply:
https://github.com/home-assistant/core/blob/dev/.github/workflows/wheels.yml#L161

Reply:
Thanks for pointing to the right file

Looks like we need `-dev` for wheels but `yaml` is needed here so I'll open a different PR for that
```
bash-5.1# lsof -p 212 -n |grep yaml
python3 212 root  mem       REG               0,33    108256      55157 /usr/lib/libyaml-0.so.2.0.9
python3 212 root  mem       REG               0,33    273824      58637 /usr/local/lib/python3.9/site-packages/yaml/_yaml.cpython-39-x86_64-linux-gnu.so
bash-5.1# apk info yaml -L
yaml-0.2.5-r0 contains:
usr/lib/libyaml-0.so.2
usr/lib/libyaml-0.so.2.0.9
```